build/packages/rpm/server/SPECS/victoriametrics.spec

```diff
@@ -12,13 +12,13 @@
 %global commit          pmm-6401-v1.82.1
 
 Name:           percona-victoriametrics
-Version:        1.82.1
+Version:        1.82.2
```

Comment:
Is this version correct? I mean the patch?

Reply:
The version does not exist. The reason I bumped the version is because of caching.
If I don't bump the version, the package is downloaded from cache and the cached version does not have the proper spec.

Reply:
I get it, but that's not the best choice - the version is still wrong. Instead, we shoud bump up the release, line 16.